var/spack/repos/builtin/packages/py-dask/package.py

```diff
@@ -38,6 +39,7 @@ class PyDask(PythonPackage):
     depends_on('python@2.7:2.8,3.5:',   type=('build', 'run'))
     depends_on('python@3.5:',           type=('build', 'run'), when='@2.0.0:')
     depends_on('python@3.6:',           type=('build', 'run'), when='@2.7.0:')
+    depends_on('python@3.7:',           type=('build', 'run'), when='@2021.4.0:')
```

Comment:
https://github.com/dask/dask/commit/1767fedd2f0662ba2fd6a9caebf2d3595ca8046c
```suggestion
    depends_on('python@3.7:',           type=('build', 'run'), when='@2021.3.1:')
```